src/test/java/seedu/watodo/testutil/TypicalTestTasks.java

```diff
@@ -32,7 +39,7 @@ public static void loadTaskManagerWithSampleData(TaskManager ab) {
     }
 
     public TestTask[] getTypicalTasks() {
-        return new TestTask[]{eat, study, sleep};
+        return new TestTask[]{code, study, eat, sleep, shop, play};
```

Comment:
I changed back to {eat, sleep, play} cos dw present all our tasks in the TestTask[]. Cos some tests want to test adding new tasks to the array